robot/Cumulus/resources/NPSP.py

```diff
@@ -678,17 +679,17 @@ def verify_date_split(self,date, no_payments, interval):
             new_date = "/".join(dates)
             mm = mm + int(interval)
             dates = list(map(str, date_list))
-            #if new_date not in t_dates: 
+            #if new_date not in t_dates:
             locator1 = npsp_lex_locators['payments']['date_loc'].format(new_date)
```

Comment:
probably rename it to date_locator or anything meaningful?

Reply:
Done